ui/pages/settings/advanced-tab/advanced-tab.component.js

```diff
@@ -631,7 +686,7 @@ export default class AdvancedTab extends PureComponent {
 
     return (
       <Box
-        ref={this.settingsRefs[11]}
+        ref={this.settingsRefs[12]}
```

Comment:
Change in numbers here is for making sure that STX will be the top notification.